tests/unit/test_consumer.py

```diff
@@ -9,6 +9,7 @@
 
 import yaml
 from charms.loki_k8s.v0.loki_push_api import LokiPushApiConsumer
+from fs.tempfs import TempFS
 from helpers import TempFolderSandbox
```

Comment:
If no one objects, I will drop `TempFolderSandbox` in favor of `TempFS` as part of this PR.